src/rapid_pro_to_engagement_db/rapid_pro_archive_client.py

```diff
@@ -23,6 +23,11 @@ def get_workspace_name(self):
             org = Org.deserialize(json.load(f))
         return org.name
 
+    def get_workspace_uuid(self):
+        with open(f"{self.archive_dir}/org.json") as f:
+            org = Org.deserialize(json.load(f))
```

Comment:
Can we have a function that returns org to avoid repetition?

Reply:
done, thanks for the suggestion